clusterman/migration/worker.py

```diff
@@ -131,4 +157,44 @@ def event_migration_worker(migration_event: MigrationEvent, worker_setup: Worker
     :param MigrationEvent migration_event: event instance
     :param WorkerSetup worker_setup: migration setup
     """
-    pass  # TODO
+    prescaling_applied = False
+    manager = PoolManager(migration_event.cluster, migration_event.pool, SUPPORTED_POOL_SCHEDULER, fetch_state=False)
+    connector = cast(KubernetesClusterConnector, manager.cluster_connector)
+    connector.set_label_selectors(migration_event.label_selectors, add_to_existing=True)
+    manager.reload_state()
+    try:
+        if worker_setup.disable_autoscaling:
```

Comment:
i'm still a little worried about us pausing the autoscaler for migrations (even with pre-scaling) - if someone triggers a migration before peak traffic times when we're scaled down, i can easily see us overwhelming the pre-scaled capacity

Reply:
(not sure if this is just a temporary thing until we get to the autoscaler interference ideas mentioned in the tech spec for this)

Reply:
This is off by default. It will just have some use in very small dev pool, the ones with 3/4 hosts in which pre-scaling by 1 causes the autoscaler to go real mad.